libpod/runtime_img.go

```diff
@@ -865,21 +865,26 @@ func (r *Runtime) RemoveImage(image *image.Image, force bool) error {
 		}
 	}
 
-	if len(image.Names()) > 1 && !force {
-		return fmt.Errorf("unable to delete %s (must force) - image is referred to in multiple tags", image.ID)
-	}
-
-	if len(image.Names()) > 1 && force {
-		// If it is forced, we have to untag the image so that it can be deleted
-		if err = r.store.SetNames(image.ID(), image.Names()[:0]); err != nil {
+	if len(image.Names()) > 1 && !image.InputIsID() {
+		// If the image has multiple reponames, we do not technically delete
+		// the image. we figure out which repotag the user is trying to refer
+		// to and untag it.
+		repoName, err := image.MatchRepoTag(image.InputName)
+		if err != nil {
+			return err
+		}
+		if err := image.UntagImage(repoName); err != nil {
 			return err
 		}
+		fmt.Println("Untagged: ", repoName)
```

Comment:
Can't do that here. libraries should not be printing to Stdout. If we need it, we should return a string (or []string? --force can remove multiple tags?)

Reply:
Could also convert this to a logrus.Infof or similar

Reply:
fixed